### PR TITLE
Site Verification: Show one-way activation banner when feature is inactive

### LINKED ISSUE #7021: Site Verification: Still visible in settings when disabled
If a user disables the 'Site Verification' module on the full modules list (/wp-admin/admin.php?page=jetpack_modules), they expect the Site Verification options to disappear from the Jetpack -> Settings -> Traffic page, but they do not.

Reported here: https://wordpress.org/support/topic/important-hidden-options-feature-toggles-and-doesnt-work-properly

### PR DESCRIPTION
**Pull request by dereksmart**
Fixes #7021

When Site Verification feature is inactive, show a one-way activation banner.  

This uses the title/long description of the module from the module header and can be manually changed if we want.  

To Test: 
- Disable site verification module
- Navigate to /traffic and view the banner
- Make sure activation works properly

Looks like this:
screen shot 2017-08-07 at 12 28 14 pm

### CODE REVIEW

File: _inc/client/traffic/verification-services.jsx [hunk 1]
```diff
@@ -16,11 +16,30 @@ import {
 import { ModuleSettingsForm as moduleSettingsForm } from 'components/module-settings/module-settings-form';
 import SettingsCard from 'components/settings-card';
 import SettingsGroup from 'components/settings-group';
+import JetpackBanner from 'components/jetpack-banner';
 
 export const VerificationServices = moduleSettingsForm(
 	React.createClass( {
+		activateVerificationTools: function() {
+			return this.props.updateOptions( { [ 'verification-tools' ]: true } );
```
Comment (eliorivero) [inline _inc/client/traffic/verification-services.jsx]: Why are brackets needed here?

File: _inc/client/traffic/verification-services.jsx [hunk 2]
```diff
@@ -16,11 +16,30 @@ import {
...
+			// Show one-way activation banner if not active
+			if ( ! this.props.getOptionValue( 'verification-tools' ) ) {
+				return (
+					<JetpackBanner
+						callToAction={ __( 'Activate' ) }
+						title={ verification.name }
+						icon="cog"
+						description={ verification.long_description }
+						onClick={ this.activateVerificationTools }
+					/>
```
Comment (eliorivero) [inline _inc/client/traffic/verification-services.jsx]: captura de pantalla 2017-08-08 a las 14 43 17

There's something odd with this, the svg is included twice.
This is not happening with other banners like the support banner at the bottom.

File: _inc/client/traffic/verification-services.jsx (hunk 1 above)
Comment (dereksmart) [inline _inc/client/traffic/verification-services.jsx]: I'm not sure, I was following the pattern of other updateOptions usage from when you refactored them

**Comment by dereksmart** [comment]
> confirm that this style is ok

I followed the same one-way activation banner styles as the ones found in search 

screen shot 2017-08-08 at 3 45 15 pm

File: _inc/client/traffic/verification-services.jsx (hunk 2 above)
Comment (dereksmart) [inline _inc/client/traffic/verification-services.jsx]: It's the same with the search banners.  
screen shot 2017-08-08 at 3 46 35 pm

Likely something going on with the `<Banner />` dops-component

Comment (dereksmart) [inline _inc/client/traffic/verification-services.jsx]: Yup, it's coming from dops-comonents and it seems like it's on purpose to include two svgs (GridIcons) here 

https://github.com/Automattic/dops-components/blob/master/client/components/banner/index.jsx#L76-L85

**Comment by dereksmart** [comment]
Added a commit that updates the dops-components to get this fix https://github.com/Automattic/dops-components/pull/112

**Comment by rickybanister** [comment]
The style looks just fine.

Just a super minor note, we removed the all-caps style from the compact buttons in Calypso, but it has not been updated in dops-components. That can happen separate from this PR, but would be good to tackle.

**Comment by zinigor** [comment]
Created https://github.com/Automattic/dops-components/pull/114 as a fix for the uppercase button styles.

**Pull request merged**